CONTRIBUTING.md

```diff
@@ -2,25 +2,27 @@
 1. Cookbook related Issues should be logged on Github's issue tracker, and are considered to be the source of truth. 
     1. Issues will be triaged and marked with an appropriate label (e.g. bug, enhancement, question).
     2. A corresponding internal JIRA could be logged if there needs to be internal discussion or if an internal code review is desired.
-2.  The issue should be worked in a branch (Outside of the cerner/cerner_splunk repo)
+2. The issue should be worked in a branch (Outside of the cerner/cerner_splunk repo)
 3. The feature is pull requested to 'stable' 
-    1. The feature branch should use as few logical commits as possible with descriptive commit messages
-    2. Attach a pull request to the issue (using a tool like hub, github's ui doesn't let you do this but the API does) if it's a simple / obvious change, or create a new pull request referencing the issue if it's larger / complex. (judgement call).
+    1. The feature branch should use as few logical commits as possible with descriptive commit messages (These commits should not reference the issue, that will be done when it is merged)
```

Comment:
I would bold or italicize the `not` portion. "These commits should _**not**_ ..."